.github/workflows/dotnetcore.yml

```diff
@@ -44,7 +44,7 @@ jobs:
     - name: Setup .NET 9.0.x
```

Comment:
Shouldn't need 6, 7, 8 with this. The 9 SDK can build 6,7,8.

Reply:
Removed.